core/src/main/java/org/stellar/anchor/event/models/TransactionEvent.java

```diff
@@ -119,6 +121,11 @@ public enum Status {
       this.status = status;
     }
 
+    @Override
```

Comment:
💡 Can we add detailed status string? It would be somewhat confusing if we see the event printed to debug messages.